components/Auth/withoutAuth.js

```diff
@@ -1,13 +1,13 @@
 import { useRouter } from "next/router";
 import { useAuth } from "./useAuth";
 import * as USER from "~/lib/utils/user";
-import LoadingLayout from "~/layouts/LoadingLayout";
+// import LoadingLayout from "~/layouts/LoadingLayout";
```

Comment:
Why remove loading effects?

Reply:
Because having a loading page like that sucks when it comes to usability. For example, with that page, this is the scenario if the login fails:

`insert data -> submit -> flicker -> go to loading page -> flicker -> back to login -> insert data again`

It's a bit cumbersome and feels unpolished.

In this case (login) the loading page was replaced by an animation in the submit button. The same will likely occur in the registration page too.

Eventually we may need some loading component, but it should be more specific and better integrated than this one.

I didn't remove it because it might be needed ¯\_(ツ)_/¯

Reply:
This animation what not happening when you did log in, but when it was checking if you were logged in, and it does not change pages, it just mounts, which is superfast and fluid. The button animation is wonderful UX, and it was already happening.

I don't think we should keep code commented like this, we should either keep it or just delete it.